uswds/uswds-site
# USWDS-Site: Fix build errors

# Summary
Fixed build errors found in https://github.com/uswds/uswds-site/pull/2096
- Updated `package-lock.json` with newest updates from `develop` to repair errors found in [a11y_mobile](https://app.circleci.com/pipelines/github/uswds/uswds-site/4977/workflows/f4e9a2c3-538a-4a2a-b5bf-244f612a2ed4/jobs/18276) tests
- Updated broken/404 links to repair errors found  during [HTML Proofer checks](https://app.circleci.com/pipelines/github/uswds/uswds-site/4977/workflows/f4e9a2c3-538a-4a2a-b5bf-244f612a2ed4/jobs/18275).

Note: HTML Proofer will still flag errors related to `https://github.com/uswds/uswds/releases/tag/v3.5.0`. This will resolve once 3.5.0 is out.

## Related PRs
This should resolve some of the build errors found in https://github.com/uswds/uswds-site/pull/2096

## Preview link

Preview link: 
- [Social security number profile pattern](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-3.5.0-build-errors/patterns/create-a-user-profile/social-security-number/#references)
- [Progress easily](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-3.5.0-build-errors/patterns/complete-a-complex-form/progress-easily/#trauma-informed-design)

## Problem statement
Outdated links were causing [404 errors](https://app.circleci.com/pipelines/github/uswds/uswds-site/4973/workflows/532c92bb-167f-476c-adc7-f0154f241a97/jobs/18253).

## Problem statement
Removed any 404 links that were not related to 3.5.0. 

Please note that `npm run proof` will still flag `https://github.com/uswds/uswds/releases/tag/v3.5.0`. This will resolve once the release is...released.

## Testing and review
- Confirm the the updated links are suitable replacements
- Confirm that `npm run proof` only flags 3.5.0 release links
